private/buf/bufformat/testdata/proto2/quotes/v1/quotes.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto2";
+
+import "google/protobuf/descriptor.proto";
+
+message Foo {
+  string something = 1;
+}
+
+extend google.protobuf.FieldOptions {
+  string name = 10001;
+  Foo foo = 10002;
+}
+
+message Foo {
+  string one = 1 [(name) = "f\"o\"'o'"];
+  string two = 2 [(name) = 'f"oo"'];
+  string three = 3 [(name) = "f'oo'"];
+  string four = 4 [(name) = "f\"o\"\'o\'"];
+  string five = 5 [(name) = "f\"o\"o"];
```

Comment:
Shouldn't we preserve the user's input here since we don't need to rewrite?

Reply:
Similar to the above, this is intentional but it is debatable.